showcase/app/templates/components/icon.hbs

```diff
@@ -0,0 +1,135 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{page-title "Icon"}}
+
+<Shw::Text::H1>Icon</Shw::Text::H1>
+
+<section data-test-percy>
```

Comment:
Is this enough for setting up tests in Percy?

Reply:
No, you have to add an entry in the file `showcase/tests/acceptance/percy-test.js`